FV3/coarse_graining/mappm.F90

```diff
@@ -0,0 +1,962 @@
+! Code here has been copied from fv_mapz.F90; this just a lightweight
+! module used for vertical interpolation in coarse_graining.F90.
```

Comment:
So nice to see  this comment explaining where the code came from!

Reply:
haha thanks to your comment I now see a typo:
```suggestion
! Code here has been copied from fv_mapz.F90; this is just a lightweight
! module used for vertical interpolation in coarse_graining.F90.
```